### ISSUE TITLE
Suggestion for hyperparameter tuning for other non-attributes datasets

### ISSUE THREAD

**Issue by Barcavin** [melifluos/subgraph-sketching#10]
Hi,

I am trying to apply ELPH/BUDDY to other non-attributed graphs (no node features attached graph). There seem to be a lot of hyperparameters to tweak. Can you provide some general suggestions about how to find good hyperparameters?

I plan to play with those non-default parameters in README on ogbl-ddi and ogbl-ppa. More advice is greatly appreciated.

**Comment by melifluos** [comment]
Hi @Barcavin we use wandb for hyperparameter tuning via sweeps. I would recommend trying it if you don't already use it. Failing that, the main hyperarameters and good ranges are
hidden dimension (128–1024), learning rate (0.0001–0.02) and dropout (0–1), weight decay (0–0.001) max_hash_hops [1,2,3].
